### PR TITLE
Add ability to post message to breakout rooms

### PR DESCRIPTION
**Pull request by marcoambrosini**
https://user-images.githubusercontent.com/26852655/215503893-7e171655-5d99-44bd-b81b-02b4adc2a503.mov

### CODE REVIEW

File: src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue [hunk 1]
```diff
@@ -99,13 +113,16 @@ import NcAppNavigationItem from '@nextcloud/vue/dist/Components/NcAppNavigationI
 import Participant from '../Participants/ParticipantsList/Participant/Participant.vue'
 import NcButton from '@nextcloud/vue/dist/Components/NcButton.js'
 import BreakoutRoomsEditor from '../../BreakoutRoomsEditor/BreakoutRoomsEditor.vue'
+import NcActionButton from '@nextcloud/vue/dist/Components/NcActionButton.js'
+import SendMessageDialog from '../../BreakoutRoomsEditor/SendMessageDialog.vue'
```
Comment (Antreesy) [inline src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue:117]: All changes seem OK for me, but different approach in different components to the organisation of imports worries me a little bit.
Maybe we could move it to separate discussion?

Comment (nickvergessen) [reply src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue:117]: Yeah, we should straighten that after the RC begin, but before we split of stable26

Comment (marcoambrosini) [reply src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue:117]: Yes, let's open an issue for that and decide what to do there!

**Review by nickvergessen — APPROVED** [review]
🐘

**Pull request merged**